composer/algorithms/seq_length_warmup/seq_length_warmup.py

```diff
@@ -97,7 +115,23 @@ class SeqLengthWarmup(Algorithm):
 
     See the :doc:`Method Card </method_cards/seq_length_warmup>` for more details.
 
-    Example: Awaiting language model test fixtures.
+    Example:
+
+    .. code-block::
```

Comment:
can this be test code?

Reply:
I filed this as a separate issue (#620) as I'm still waiting on language model test fixtures.